package exception;

public class Exception01 {
	public static void main(String[] args) {
		System.out.println("100 ÷ 0 は？");
		int reslt = 100 / 0;
		System.out.println("計算結果" + reslt);
		System.out.println("プログラムを終了します");
	}
}
